### DISCUSSION TITLE
Suggestion for authentication

### DISCUSSION THREAD

**Discussion by jdenka** [M365-Consultant/EntraID-MFA-DynamicGroup#1] [Ideas]
I have a small suggestion regarding the authentication part, you could change it so you only use Connect-MgGraph if you are using Microsoft.Graph modules above version 2.0 since they have added support for Managed Identities then. 

You can check it out here: https://learn.microsoft.com/en-us/powershell/microsoftgraph/authentication-commands?view=graph-powershell-1.0#using-managed-identity

**Comment by M365-Consultant** [comment]
Thank you Dennis for this great input. I've changed it and the test run was successful. Already updated the code here as well 👍
